### PR TITLE
Subscribe to all events in the test dApp

### PR DESCRIPTION
**Pull request by AlirezaHaghshenas**
Related to: #2707 

Thank you for your contribution to Taquito.

Before submitting this PR, please make sure:

- [ ] Your code builds cleanly without any errors or warnings
- [ ] You have run the linter against the changes
- [ ] You have added unit tests (if relevant/appropriate)
- [ ] You have added integration tests (if relevant/appropriate)
- [ ] All public methods or types have TypeDoc coverage with a complete description, and ideally an @example
- [ ] You have added or updated corresponding documentation
- [ ] If relevant, you have written a first draft summary describing the change for inclusion in Release Notes. 

In this PR, please also make sure: 

- [ ] You have linked this PR to the issue by putting `closes #TICKETNUMBER` in the description box (when applicable)
- [ ] You have added a concise description on your changes
## Release Note Draft Snippet

In the test dApp, all beacon events are now subscribed to and shown in the UI

### CODE REVIEW

File: apps/taquito-test-dapp/vite.config.ts [hunk 1]
```diff
@@ -29,6 +29,13 @@ export default ({ command }) => {
           }/index.js`
           // `../../node_modules/@airgap/beacon-dapp/dist/walletbeacon.dapp.min.js`
         ),
+        "@airgap/beacon-sdk": path.resolve(
+          path.resolve(),
+          // "./src/walletbeacon.dapp.min.js"
```
Comment (dsawali) [inline apps/taquito-test-dapp/vite.config.ts:34]: are these comments needed? If not then it's probably better to clean up these artifacts

File: apps/taquito-test-dapp/src/config.ts [hunk 2]
```diff
@@ -1,14 +1,31 @@
...
+}
+
+export const getTzKtUrl = (networkType: SupportedNetworks): string | undefined => {
+  switch (networkType) {
+    case NetworkType.NAIROBINET:
+      return "https://nairobinet.tzkt.io";
+    case NetworkType.GHOSTNET:
+      return "https://ghostnet.tzkt.io";
+    case NetworkType.MAINNET:
+      return "https://tzkt.io";
```
Comment (hui-an-yang) [inline apps/taquito-test-dapp/src/config.ts:23]: preferably using ecad mainnet node url `https://mainnet.ecadinfra.com`?

File: apps/taquito-test-dapp/src/Layout.svelte [hunk 3]
```diff
@@ -1,6 +1,5 @@
 <script lang="ts">
   import Sidebar from "./lib/Sidebar.svelte";
-  import TaquitoLogo from "./lib/TaquitoLogo.svelte";
```
Comment (hui-an-yang) [inline apps/taquito-test-dapp/src/Layout.svelte:3]: possible to still have a static log somewhere in the background?

File: apps/taquito-test-dapp/src/lib/TestContainer.svelte [hunk 4]
```diff
@@ -335,36 +335,12 @@
...
-                  rel="noopener noreferrer"
-                >
-                  {shortenHash(opHash)}
-                </a>
-              {:else if $store.networkType === NetworkType.MAINNET}
-                <a href={`https://tzkt.io/${opHash}`} target="_blank" rel="noopener noreferrer">
+              {#if $store.networkType === NetworkType.CUSTOM}
+                {shortenHash(opHash)}
+              {:else}
+                <a href={`${getTzKtUrl($store.networkType)}/${opHash}`} target="_blank" rel="noopener noreferrer">
```
Comment (hui-an-yang) [inline apps/taquito-test-dapp/src/lib/TestContainer.svelte:341]: Curious of the reason to change from better-call.dev to tzkt

Comment (AlirezaHaghshenas) [reply apps/taquito-test-dapp/src/lib/TestContainer.svelte:341]: AFAIK TzKt is the newer, more feature rick, more actively developed successor to BCD.

File: apps/taquito-test-dapp/src/Layout.svelte (hunk 3 above)
Comment (AlirezaHaghshenas) [reply apps/taquito-test-dapp/src/Layout.svelte:3]: I think it's OK to remove it for now. I've started a doc to improve the test dApp in notion

**Review by dsawali — APPROVED** [review]

**Pull request merged**